deployments/k8s-v1.16/sriovdp-daemonset.yaml

```diff
@@ -111,13 +116,18 @@ spec:
           mountPath: /var/log
         - name: config-volume
           mountPath: /etc/pcidp
+        - name: device-info
+          mountPath: /var/run/k8s.cni.cncf.io/devinfo/dp
       volumes:
         - name: devicesock
           hostPath:
             path: /var/lib/kubelet/
         - name: log
           hostPath:
             path: /var/log
+        - name: device-info
+          hostPath:
```

Comment:
do you need to specify
`DirectoryOrCreate` type ?

what if the path does not exist on the host ? will it be created ?

https://kubernetes.io/docs/concepts/storage/volumes/#hostpath

Reply:
I did not see any error when I tested it. The nad library [would take care of creating the directory](https://github.com/k8snetworkplumbingwg/network-attachment-definition-client/blob/9d213757d22d5d06f1b8ff289ce16d0cf9aabc6c/pkg/utils/cniconfig.go#L163) so from a DevicePlugin perspective, it should not care. However, it's not a bad idea to add that type flag.
Thanks